FIAT/finite_element.py

```diff
@@ -60,6 +60,11 @@ def entity_closure_dofs(self):
         freedom on the closure of those entities for the finite element."""
         return self.dual.get_entity_closure_ids()
 
+    def entity_permutations(self):
+        """Return the map of topological entities to the map of orientations
+        to permutation lists for the finite element."""
```

Comment:
Can you just paste the dualset docstring here?

```
entity_permutations.__doc__ = DualSet.get_entity_permutations.__doc__
```

Reply:
Done.